translate.php

```diff
@@ -12,7 +12,7 @@
 
 can't be null if $strict is set to true
 */
-$use_lang='de';
+$use_lang='en';
```

Comment:
Die Webseite ist eigentlich ursprünglich in Deutsch.

Reply:
Und es werden wahrscheinlich auch Leute anschauen, deren Browser u.U. IE ist 😒 
Dann könnte es außerdem auch so vorkommen, dass
- der Header nicht übermittelt wird (warum auch immer)
- der Browser einfach falsch eingestellt ist und der Benutzer es nicht besser weiß.